### ISSUE TITLE
Importing Cellfinder data into Clean Counts pipeline tutorial

### ISSUE THREAD

**Issue by JuiceSackey** [valeriabonapersona/abc4d#1]
Hey, Im trying to run the abc4d.npy exported file from cellfinder through this program and I am running into an issue with the coordinates. Looking at the example dataset, it seems all the coordinates are taken from the atlas and left and right is defined in cleaning_functions as such (zPos-228 to find midline). However, in my exported file the coordinates can range up to a scale of 10k+, and very clearly different from the example dataset. Not sure if this is cellfinder issue or what. But all my cells are labeled as being in the right hemisphere, and im sure the cleaning is also wrong if the coordinates are messed up. Could you take a look at the export and let me know if this is right or not, thanks!

<img width="450" alt="Screenshot 2021-09-28 142310" src="https://user-images.githubusercontent.com/54648048/135152769-8444e199-1536-4695-a327-173a2a25ad31.png">

The abc4d.npy file is attached below:
[abc4d_2.npy](https://wustl.box.com/s/b574uwt9cknquyep4opw6ve46nipfycs)

**Comment by valeriabonapersona** [comment]
Hi! Just checking - which atlas did you use? so far I only have the Allen Brain Mouse Atlas from 2017. If you have not used this atlas, there are a couple of things to adapt in abc4d (not implemented yet) before using it!

**Comment by JuiceSackey** [comment]
Yes, I am using the Allen Mouse Brain Atlas at 25 um resolution. Im thinking maybe the coordinates were just exported in a different format from that in the example dataset or something

**Comment by valeriabonapersona** [comment]
Is the file exported by cellFinder in 32 or 64 bits? Should be 64

**Comment by valeriabonapersona** [comment]
Hi! I had a look, and actually yPos looks good on my computer (see below). My guess is that maybe your R is a 32bits rather than 64? I am asking because initially cellFinder was exporting in 32bits, but (my) R was 64 and I was having a similar issue. 
<img width="108" alt="Screenshot 2021-10-01 at 18 52 42" src="https://user-images.githubusercontent.com/46342235/135658750-b5b78f65-0727-484d-a96c-540c60359e57.png">

**Comment by JuiceSackey** [comment]
Mine looks the exact same as yours: 
<img width="226" alt="xpos" src="https://user-images.githubusercontent.com/54648048/135676724-49d614a8-b8f5-4b3d-8aec-c495f37c20ab.png">
 However when you scroll down the yPos coordinates get just as high. I checked and I am using R 4.1 [64-bit], with Rstudio 1.417.

**Comment by valeriabonapersona** [comment]
Ah I see - sorry I should have seen that. As you suggested, it has to do with what cellFinder exports. Unfortunately I can't do anything about it, but I will email Adam and I am sure he will have ideas.

For now - you should have problems only with 1) right/left (due to the definition in the code) and 2) the masks corrections (since these are also based on the ABA coordinates). If you want to continue with your code, you can manually change the definition of midline, and leave the masks' df empty (although they need to have the same col names). The rest should work! I will keep you updated for the export issue.

**Comment by valeriabonapersona** [comment]
Ah I see - sorry I should have seen that. As you suggested, it has to do with what cellFinder exports. Unfortunately I can't do anything about it, but I will email Adam and I am sure he will have ideas.

For now - you should have problems only with 1) right/left (due to the definition in the code) and 2) the masks corrections (since these are also based on the ABA coordinates). If you want to continue with your code, you can manually change the definition of midline, and leave the masks' df empty (although they need to have the same col names). The rest should work! I will keep you updated for the export issue.

**Comment by JuiceSackey** [comment]
Ah okay, for the time being, I will try your suggestions. Thanks again!

**Comment by valeriabonapersona** [comment]
Update: it may be that cellFinder outputs in µm rather than in slices (as in abc4d and ABA). Therefore, each coordinate would be slices*25µm. The coordinates in your df always increase in steps of 25, and when I divided them, the values were similar to our example dataset. 

I have contacted cellFinder to be sure :)

**Comment by JuiceSackey** [comment]
Thanks a bunch! This makes it much easier
